calico-enterprise_versioned_docs/version-3.18/releases.json

```diff
@@ -1,259 +1,263 @@
 [
   {
-    "title": "master",
+    "title": "v.3.18.0",
```

Comment:
Bad find & replace: `"v.3.18.0"` should be `v3.18.0`

Reply:
Fixed.